FRAuth/FRAuth/User/Browser.swift

```diff
@@ -316,7 +316,7 @@ import SafariServices
         var viewController: SFSafariViewController?
         if #available(iOS 11.0, *) {
             let config = SFSafariViewController.Configuration()
-            config.entersReaderIfAvailable = true
+            config.entersReaderIfAvailable = false
             viewController = SFSafariViewController(url: url, configuration: config)
             viewController?.delegate = self
         }
```

Comment:
false is the default value. I wonder if we should still explicitly set it. Also as we are changing, maybe we should get rid of the iOS 11 check as we support iOS 12+. So I suggest to replace lines 317-326 with the following:

        viewController = SFSafariViewController(url: url, configuration: SFSafariViewController.Configuration())
        viewController?.delegate = self